src/index.ts

```diff
@@ -140,9 +140,10 @@ export class SmartsheetConnector extends CoreConnector {
         ).catch(console.error)
         res.send('')
       }
+      return true
+    } else {
```

Comment:
else isn't necessary

Reply:
just being picky here

Reply:
It is true that it isn't necessary- I just like this code type personally but I don't mind either way here :)